automation/project/Dependencies.scala

```diff
@@ -5,14 +5,14 @@ object Dependencies {
 
   val akkaV         = "2.6.8"
   val akkaHttpV     = "10.2.0"
-  val jacksonV      = "2.14.1"
+  val jacksonV      = "2.15.2"
```

Comment:
Needed to update Jackson because workbenchLibs updated to this version, and we were getting mismatches in automation:

```
Caused by: com.fasterxml.jackson.databind.JsonMappingException: Scala module 2.14.1 requires Jackson Databind version >= 2.14.0 and < 2.15.0 - Found jackson-databind version 2.15.2